index.html

```diff
@@ -4,10 +4,106 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Lesson-3</title>
 </head>
 <body>
-    <h1>Моя первая страница</h1>
+    <h1>Моя первая &#128516; &#128525; страница</h1>
     <p><a href="./about/index.html">Об авторе</a></p>
+
+    <p>Всё началось в не так &#9749 далёком <strong>2018 году</strong> когда я начал служить в наших <em>доблестных пограничных войсках.</em><br>Тогда я как обычно сидел в <mark>youtube</mark> и случайно наткнулся на обучающее видео по HTML. Тогда я его просто просмотрел и немного стало интересно а как это работает.<br> После пробовал самостоятельно учить всё это но информации <mark>для изучения стало слишком много</mark> и я решил пойти куда-нибудь учиться.<br> Я купил курс на FructCode там показывали как делается макет сайта с фильмами. шло время, а я так толком ничему и не научился, плюс на работе было просто не до учёбы и я забил на это всё.</p>
+    <p><ins>Прошлой зимой я уволился из ОПС &#10004</ins> и начал работать тренером по л/атлетике. З/п у тренера очень маленькая и я решил пойти учиться куда-нибуть чтоб в последующем устроиться на нормальную работу.<br> Вот тут и вспомнил про все то что учил когда-то и решил дальше двигаться в этом направлении и чем глубже я погружаюсь тем мне всё более интересно как это работает и как решать все новые более сложные задачи.</p>
+    
+    <p><img src="/ghtrfbhj.jpg" alt="Программист"></p>
```

Comment:
This relative url is not working. We use '/' only for folders. In this case we can use './' or just 'ghtrfbhj.jpg'. But I would recommend store all images separately in folder. For example create folder "img" and put all images inside